sdk/src/server-api/sc/framework/plugins/AbstractGame.kt

```diff
@@ -130,17 +130,16 @@ abstract class AbstractGame<P : Player>(override val pluginUUID: String) : IGame
      * - sends out a state update and requests a new move
      */
     protected fun next(nextPlayer: P?) {
+        // if paused, notify observers only (so they can update the GUI appropriately)
+        notifyOnNewState(currentState, isPaused)
```

Comment:
are these only listeners with admin privileges which are notified here or could my client register itself as an observer and receive the new state on a paused game earlier?

Reply:
this didn't change anything regarding that, only that it is now also sent before game over

Reply:
Yes, observation is only possible with admin rights, checked in sc/server/Lobby.kt:75